CONTRIBUTING.md

```diff
@@ -2,13 +2,33 @@
 
 ## Table of Contents
 - [Claiming an Issue](#Claiming-an-Issue)
+- [step 1. create an issue](#step-1.-create-an-issue)
+- [step 2 fork the code](#step-2-fork-the-code)
+- [step 3 Make the necessary changes](#step-3-Make-the-necessary-changes)
+- [step 4 Submit a PR](#step-4-Submit-a-PR)
+- [CONGRADUALTIONS](#CONGRADUALTIONS)
 
-## Claiming an Issue [Back to Top](#Table-of-Contents)
-
+# Claiming an Issue [Back to Top](#Table-of-Contents)
```

Comment:
This wasn't the suggestion I was making....

instead of h1 this should be an H3 (###)

```suggestion
### Claiming an Issue [Back to Top](#Table-of-Contents)
```